test/Microsoft.AspNetCore.Mvc.RazorPages.Test/project.json

```diff
@@ -1,14 +1,19 @@
 ﻿{
   "buildOptions": {
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": {
+      "include": "../Microsoft.AspNetCore.Mvc.TestCommon/TestFile*.cs"
+    }
```

Comment:
I think I had some odd issues referencing TestCommon before we had rebased. I'll see if it's fixed now.